resources/etc/default/openhab2

```diff
@@ -29,3 +29,8 @@ EXTRA_JAVA_OPTS=""
 #
 #OPENHAB_USER=openhab
 #OPENHAB_GROUP=openhab
+#
+# The startmode for the openHAB runtime. Only available for systemctl/systemd systems.
+# Defaults to daemon when unset here.
```

Comment:
I wasn't so sure, options for karaf are:

```
clean   - Deletes $KARAF_DATA contents
debug   - Turns on java debugging options
status  - Checks if running
stop    - Stops instance
console - ???
server  - Starts remote shell and runs the karaf server
daemon  - Same as server, extra exec and return details.
client  - Starts the client to an existing instance
```

Out of them, we could really only recommend using `daemon`, `debug` or `debug daemon` with systemd.